src/app.ts

```diff
@@ -296,6 +303,16 @@ async function runBuild(input: string, program: OptionValues, helpText: string)
     process.exit(1);
   }
 
+  // UNCOMMENT then following lines to get a SQLite export of the FPL database for debugging
+  //
+  // const fplExport = await defs.exportDB();
+  // if (fplExport.mimeType === 'application/x-sqlite3') {
+  //   const exportPath = path.join(outDir, 'fsh-generated', 'FPL.sqlite');
+  //   fs.ensureDirSync(path.join(outDir, 'fsh-generated'));
+  //   fs.writeFileSync(exportPath, fplExport.data);
+  //   logger.info(`Exported FPL database to ${exportPath}`);
+  // }
+
```

Comment:
We could do that, but I am a little concerned about complicating options by adding something that is not useful to general users (and may confuse them). Right now our options are:
```
Options:
  -l, --log-level <level>  specify the level of log messages (default: "info") (choices: "error", "warn", "info",
                           "debug")
  -o, --out <out>          the path to the output folder (default: "fsh-generated")
  -p, --preprocessed       output FSH produced by preprocessing steps
  -r, --require-latest     exit with error if this is not the latest version of SUSHI (default: false)
  -s, --snapshot           generate snapshot in Structure Definition output (default: false)
  -c, --config <config>    override elements in sushi-config.yaml (supported: 'version', 'status', 'releaselabel') (eg:
                           --config status:draft)
  -h, --help               display help for command
```
These all make sense as options a standard user might use productively. Exporting a SQLite DB is more of a developer feature -- but since a developer is generally already in the code anyway, uncommenting isn't a super big deal. (And even for developers, the times they need a SQLite export will likely be _very rare_). I'm just not sure we should elevate something that niche to the user-facing options.

Reply:
I agree that this is definitely more of a developer feature than a user feature. Okay, let's leave it commented out.